mappings/net/minecraft/entity/EntityStatus.mapping

```diff
@@ -0,0 +1 @@
+CLASS aie net/minecraft/entity/EntityStatus
```

Comment:
Since this class is only used to compute the size of entities and rendering them, maybe call it `Pose` ?